course/test/src/isometry_TEST.cc

```diff
@@ -21,8 +22,8 @@ GTEST_TEST(Vector3Test, Vector3Operations) {
   const Vector3 p{1., 2., 3.};
   const Vector3 q{4., 5., 6.};
 
-  EXPECT_EQ(p + q, {5., 7., 9.});
-  EXPECT_EQ(p - q, {-3., -3., -3.});
+  //EXPECT_EQ(p + q, {5., 7., 9.});
```

Comment:
GTESTs macro has a problem  with pure initializer lists that causes this not to build, which I guess is the reason you commented out this. Make it `std::initializer_list({5., 7., 9.})`.